docs/glossary.md

```diff
@@ -0,0 +1,194 @@
+---
+id: glossary
+title: Glossary
+sidebar_label: Glossary
+description: "A glossary of essential terms, definitions, and concepts related to Avail."
+
+keywords:
+    - docs
+    - avail
+    - Glossary
+    - Monolithic
+    - data availability
+image: https://availproject.github.io/img/avail/AvailDocs.png
+
+
+
+---
+
+
+
+## General
+
+### Monolithic Blockchain
+A modular blockchain is one that focuses on handling a select few duties and outsources the rest to one or more separate layers.
+
+### Modular Blockchain 
+Monolithic blockchains is a blockchain which execute all of the core functions of a blockchain (Execution, Settlement, Ordering, Data Availability) inside a single blockchain
+### Data Availability (DA)
+Data Availability is the ability of nodes to access and validate the data associated with a blockchain.
+### Scalability
+Scalability in Avail refers to the ability to increase the amount of data published by the chain without negatively impacting the participants and users of the chain. This is achieved through a modular design that separates data availability (DA) from execution. This allows for modular scaling of key constructs, as they can be optimized individually.
+
+### Execution
+Execution is how nodes on the blockchain process transactions to transition the blockchain between states. Avail uses optimistic and succinct systems (zk) to scale execution. This means that transactions can be executed off-chain and then verified on-chain using proofs of execution. This allows Avail to handle a large number of transactions without sacrificing security or decentralization.
+
+### Settlement
+The "finality" that blockchains offer is a promise that the transactions that have been recorded in the chain's history are unchangeable (or "immutable"). The blockchain must be persuaded of a transaction's authenticity for this to occur. The chain must therefore validate transactions, check the accuracy of proofs, and resolve disagreements in order to perform the settlement function.
+
+### Decoupling
+The concept behind a modular blockchain is that it can focus on a few tasks rather than attempting to perform all of them. Decoupling refers to the process of separating functions into modular layers or components.
+
+### Bridging
+Data Attestation Bridge (still in its testnet phase) relays the attested Merkle root from Avail to a specific bridge contract on Ethereum, serving as proof that the Avail validators have reached consensus on the data's availability.
```

Comment:
To make this definition more intuitive, let’s begin with a general statement about the bridge—it acts as a communication mechanism between Avail and Ethereum, performing specific functions such as xyz. Following this, we can go into specifics, illustrating how the Data Attestation Bridge, currently in its testnet phase, relays the attested Merkle root from Avail to a designated bridge contract on Ethereum.